agent/config/config_test.go

```diff
@@ -37,12 +37,14 @@ func TestMerge(t *testing.T) {
 	conf1 := &Config{Cluster: "Foo"}
 	conf2 := Config{Cluster: "ignored", APIEndpoint: "Bar"}
 	conf3 := Config{AWSRegion: "us-west-2"}
+	conf4 := Config{PortRange: "100-200"}
```

Comment:
you don't need to modify this unit test since you are not changing the merge method